config/crd/bases/activemonitor.keikoproj.io_healthchecks.yaml

```diff
@@ -390,10 +390,10 @@ spec:
           properties:
             description:
               type: string
+            level:
+              type: string
             repeatAfterSec:
               type: integer
-            scope:
-              type: string
```

Comment:
Breaking change?

Reply:
generally, no... it may be the case that some `HealthChecks` listed this attribute in their spec BUT it had no effect eitherway. So, in the worst case, if there are broken `HealthChecks`, they'll be able to easily be rectified by removing this attribute in the spec (an optional attr)